go/arrow/go.sum

```diff
@@ -1,3 +1,4 @@
+github.com/apache/arrow v0.0.0-20190420091023-b38936f08bcf h1:9EikFTVbwn2PyLkRdCwzya3TLwDKZwjBrnl4Hg0m7Dw=
```

Comment:
that doesn't sound right. `github.com/apache/arrow` isn't a Go module.

how did you end up with this line?